doc/source/roadmap-detailed.rst

```diff
@@ -65,32 +65,32 @@ Making this easier is a priority.  In addition, we should run them in our CI
 (gh-8779 is an ongoing attempt at this).
 
 
-Other
-`````
-
+Use of Cython
+`````````````
 Regarding Cython code:
 
 - It's not clear how much functionality can be Cythonized without making the
   .so files too large.  This needs measuring.
 - Cython's old syntax for using NumPy arrays should be removed and replaced
   with Cython memoryviews.
 
-Regarding build environments:
 
-- SciPy builds from source on Windows now with a MSVC + MinGW-w64 gfortran
-  toolchain, which we're using for official releases.
-  MSVC + Intel Fortran + MKL works as well, and is easier for users (as long
-  as they have access to ifort and MKL of course).  This mainly needs better
-  documentation at the moment.
-- We're aiming to gradually increase the minimum version of LAPACK that is
-  required, so we can use newer features.  Support for Accelerate on macOS
-  has been dropped.  We do rely quite heavily on OpenBLAS, and its stability
-  is a worry (often only one of the recent releases works without test
-  failures) - improvements in testing and build documentation at least are
-  needed.
+Windows build issues
+````````````````````
+SciPy critically relies on Fortran code. This is still problematic on Windows.
+There are currently only two options: using Intel Fortran, or using MSVC +
+gfortran.  The former is expensive, while the latter works (it's what we use
+for releases) but is quite hard to do correctly.  For allowing contributors and
+end users to reliably build SciPy on Windows, using the Flang compiler looks
+like the best way forward long-term.  Until Flang support materializes, we need
+to streamline and better document the MSVC + gfortran build.
```

Comment:
This has been tested with around 10 users already. Looks like the existing documentation is actually valid.

Reply:
I agree. I think it can be moved to the top level of the contributor guide instead of being hidden where it is.

Reply:
Thanks, removed.